lib/rails-settings/configuration.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RailsSettings
+  InvalidConfigurationError = Class.new(StandardError)
+
+  class Configuration
+    attr_reader :storage
+
+    def initialize
+      @storage = Rails.cache
+    end
+
+    def storage=(value)
+      unless value.is_a?(ActiveSupport::Cache::Store)
+        raise InvalidConfigurationError, <<~TXT
+          Option `storage` must be an instance of `ActiveSupport::Cache::Store` class.
+        TXT
+      end
+
+      @storage = value
+    end
+  end
+
+  class << self
+    def configuration
```

Comment:
Rename to `config`, like Rails or other RubyGems.

Reply:
https://github.com/huacnlee/rucaptcha/blob/main/lib/rucaptcha/configuration.rb
https://github.com/huacnlee/rucaptcha/blob/main/lib/rucaptcha.rb#L23

Please follow this code to define a configuration.

Reply:
Done